README.md

```diff
@@ -270,52 +271,50 @@ Alternatively, you can install each controller stack into a unique namespace (re
 - The organization level
 - The enterprise level
 
-There are two ways to use this controller:
+Runners can be deployed as 1 of 2 abstractions: 
 
-- Manage runners one by one with `Runner`.
-- Manage a set of runners with `RunnerDeployment`.
+- A `RunnerDeployment` (based on k8s's `Deployments`)
```

Comment:
Nit but- `RunnerDeployment`'s API and implementation doesn't exactly follow/inherit Deployment's.
Perhaps something like the below would better summarize it?

```suggestion
- A `RunnerDeployment` (similar to k8s's `Deployments`, based on `Pods`)
```